src/main/kotlin/me/partlysanestudios/partlysaneskies/api/events/PssEvents.kt

```diff
@@ -0,0 +1,58 @@
+package me.partlysanestudios.partlysaneskies.api.events
+
+import org.apache.logging.log4j.LogManager
+import java.lang.reflect.Method
+
+
+object PssEvents {
```

Comment:
Any chance we can change to PSSEventManager? Although might get confused with EventsHandler so if not just PSSEvents

Reply:
I dont think changing it to PSSEventManager would be the best, this class is only be used to register and for 1 function in the event to post, no other usages of this object should be present anywhere else.